aws-geo-location/src/androidTest/java/com/amplifyframework/geo/location/SearchApiTest.kt

```diff
@@ -129,7 +129,7 @@ class SearchApiTest {
 
         // First entry is on top of originally queried coordinates (within 1km)
```

Comment:
change comment to match new tolerance

Reply:
The comment is still correct, the tolerance is 1 km. I adjusted the value used in the inequality to be consistent with checking the distance is within 1 km.

Reply:
ohhh I see, 1km is the distance and the .0002 is the tolerance then?  maybe I should've read the part that says 'angle'...